### PR TITLE
Update To LIVD table

### PR DESCRIPTION
**Pull request by jorg3lopez**
This PR updates the LIVD table.

Test Steps:
1. Run ./gradlew
   - test
   - testIntegration
   - testEnd2End 

## Changes
- Uploaded the newest LIVD table and renamed it to LIVD-SARs-CoV-2-lastes.csv
- Renamed the old LIVD table to LIVD-SARs-CoV-2-old-1.csv
- Ran generatedocs
- Verified values against the supplemental table to see if they were unchanged (no changes)

### Testing
- [x] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Fixes
- #2912

### CODE REVIEW

**Comment by jorg3lopez** [comment]
Working to fix some of the test cases.

File: prime-router/build.gradle.kts [hunk 1]
```diff
@@ -109,6 +109,7 @@ tasks.test {
 
     // Set max parellel forks as recommended in https://docs.gradle.org/current/userguide/performance.html
     maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
+    setForkEvery(1)
```
Comment (carlosfelix2) [inline prime-router/build.gradle.kts:112]: What does this do?  Per documentation:
By default, Gradle automatically uses a separate JVM when executing tests, so changing this property is usually not necessary.

File: prime-router/metadata/tables/LIVD-SARS-CoV-2-old-1.csv [hunk 2]
```diff
@@ -0,0 +1,4713 @@
+Manufacturer,Model,Vendor Analyte Name,Vendor Specimen Description,Vendor Result Description,Test Performed LOINC Code,Test Performed LOINC Long Name,Test Ordered LOINC Code,Test Ordered LOINC Long Name,Vendor Comment,Vendor Analyte Code,Vendor Reference ID,Testkit Name ID,Testkit Name ID Type,Equipment UID,Equipment UID Type,Component,Property,Time,System,Scale,Method,Publication Version ID,LOINC Version ID
```
Comment (carlosfelix2) [inline prime-router/metadata/tables/LIVD-SARS-CoV-2-old-1.csv:1]: From the name, it sounds like this file should be deleted.  All other tables have a date.  I do not think this should be checked in

File: prime-router/metadata/tables/LIVD-SARS-CoV-2-2021-10-27.csv [hunk 3]
```diff
@@ -0,0 +1,4819 @@
+Manufacturer,Model,Vendor Analyte Name,Vendor Specimen Description,Vendor Result Description,Test Performed LOINC Code,Test Performed LOINC Long Name,Test Ordered LOINC Code,Test Ordered LOINC Long Name,Vendor Comment,Vendor Analyte Code,Vendor Reference ID,Testkit Name ID,Testkit Name ID Type,Equipment UID,Equipment UID Type,Component,Property,Time,System,Scale,Method,Publication Version ID,LOINC Version ID
```
Comment (carlosfelix2) [inline prime-router/metadata/tables/LIVD-SARS-CoV-2-2021-10-27.csv:1]: Is this the same as the latest table?  if it is then I suggest not having duplicate files.

File: prime-router/src/test/kotlin/serializers/Hl7SerializerTests.kt [hunk 4]
```diff
@@ -94,21 +95,21 @@ OBX|5|CWE|95419-8^Has symptoms related to condition of interest^LN^^^^2.69||N^No
 SPM|1|||258500001^Nasopharyngeal swab^SCT||||71836000^Nasopharyngeal structure (body structure)^SCT^^^^2020-09-01|||||||||202102090000-0600^202102090000-0600
 NTE|1|L|This is a final comment|RE"""
     }
-
+    @Ignore
```
Comment (carlosfelix2) [inline prime-router/src/test/kotlin/serializers/Hl7SerializerTests.kt]: Why are these tests being ignored?

**Review by carlosfelix2 — CHANGES_REQUESTED** [review]

**Comment by jorg3lopez** [comment]
Blocked: We are still testing an issue were one of the tests hangs ( `test incorrect HL7 content`). This causes the heap to run out of memory and it crashes. @RickHawesUSDS  is helping me out on it, as I had to switch to state support. Right now, it will not hang as I'm forking a jvm per test case. This is only for testing purposes, and this branch should not be merge as it stands right now.

**Comment by jorg3lopez** [comment]
Will close it until I we figure out what's happening with the hanging test case.

**Pull request closed without merging**